tests/zero-copy/programs/zero-copy/src/lib.rs

```diff
@@ -97,7 +97,7 @@ pub struct CreateBar<'info> {
         init,
         seeds = [authority.key().as_ref(), foo.key().as_ref()],
         bump,
-        payer = authority, owner = *program_id,
+        payer = authority, owner = ID(),
```

Comment:
I had to update this test.
It was using the `try_accounts`'s `program_id` parameter which is now `__program_id`.
I think using `ID()` is cleaner.